### PR TITLE
Feature/io t conect sdk 2.0

### PR DESCRIPTION
**Pull request by sw-tt-kirtanvaghela**

### CODE REVIEW

File: IoTConnect/IoTConnect/Info.plist [hunk 1]
```diff
@@ -16,7 +16,10 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
-	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>NSAppTransportSecurity</key>
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/Info.plist:19]: Why you need this?

File: IoTConnect/IoTConnect/IoTConnect.h [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+//
+//  Created by kirtan.vaghela on 09/01/24.
+//
 
 #import <Foundation/Foundation.h>
 
-#import "GCDAsyncSocket.h"
-#import "GCDAsyncUdpSocket.h"
-
+//#import "MqttCocoaAsyncSocket.h"
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/IoTConnect.h]: remove unwanted code

File: IoTConnect/IoTConnect/IoTConnectManager/IoTConnectService.swift [hunk 3]
```diff
@@ -466,13 +483,18 @@ extension IoTConnectManager {
...
     private func checkForIsValidOrNotWith(forData dictForData: [String: Any], withValue idValue: Any) -> Bool {
         if dictForData["dt"] as? Int == DataType.DTNumber {
-            let scan = Scanner(string: "\(Int(String(describing: idValue)) ?? 0)")
-            var val: Int32 = 0
-            if scan.scanInt32(&val) && scan.isAtEnd {
-                
-            } else {
-                return false
-            }
+//            let scan = Scanner(string: "\(Int(String(describing: idValue)) ?? 0)")
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/IoTConnectManager/IoTConnectService.swift]: remove unwanted code

File: IoTConnect/IoTConnect/IoTConnectSDK.h [hunk 4]
```diff
@@ -0,0 +1,22 @@
+//
+//  IoTConnectSDK.h
+//  IoTConnectSDK
+//
+//  Created by kirtan.vaghela on 29/12/23.
+//
+
+#import <Foundation/Foundation.h>
+
+//#import "MqttCocoaAsyncSocket.h"
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/IoTConnectSDK.h]: remove unwanted code

File: IoTConnect/IoTConnect/MQTTClient.swift [hunk 5]
```diff
@@ -45,9 +47,9 @@ class MQTTClient {
         mqtt!.username = dataSyncResponse[keyPath:"p.un"] as? String
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/MQTTClient.swift]: please rewrite this, remove explicit unwrap

File: IoTConnect/IoTConnect/Utilities/MQTT copy/CocoaMQTT.h [hunk 6]
```diff
@@ -0,0 +1,17 @@
+//
```
Comment (sw-ot-deveshmevada) [inline IoTConnect/IoTConnect/Utilities/MQTT copy/CocoaMQTT.h]: Why two folder are added "MQTT copy" and "MQTT"?

File: IoTConnect2/IoTConnect2/IoTConnect_2.0/Info.plist [hunk 7]
```diff
@@ -18,5 +18,10 @@
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>NSAppTransportSecurity</key>
```
Comment (sw-ot-deveshmevada) [inline IoTConnect2/IoTConnect2/IoTConnect_2.0/Info.plist:21]: Why we need this one?

**Review by sw-ot-deveshmevada — CHANGES_REQUESTED** [review]
I see There are multiple folder and many files renamed please review this pull request careful and update things

File: IoTConnect/IoTConnect/Info.plist (hunk 1 above)
Comment (sw-tt-kirtanvaghela) [reply IoTConnect/IoTConnect/Info.plist:19]: There was a HTTP URL for api

File: IoTConnect/IoTConnect/Utilities/MQTT copy/CocoaMQTT.h (hunk 6 above)
Comment (sw-tt-kirtanvaghela) [reply IoTConnect/IoTConnect/Utilities/MQTT copy/CocoaMQTT.h]: Deleted MQTT copy

File: IoTConnect2/IoTConnect2/IoTConnect_2.0/Info.plist (hunk 7 above)
Comment (sw-tt-kirtanvaghela) [reply IoTConnect2/IoTConnect2/IoTConnect_2.0/Info.plist:21]: There was a HTTP URL for api

File: IoTConnect/IoTConnect/Info.plist (hunk 1 above)
Comment (sw-ot-deveshmevada) [reply IoTConnect/IoTConnect/Info.plist:19]: this need to be conveyed to IoTConnect this is not allowed, I am resolving for now

**Comment by sw-ot-deveshmevada** [comment]
@sw-tt-kirtanvaghela @sw-tt-rumitkkhakhi  Where is the IoTConnect 1.0 code?

**Comment by sw-tt-kirtanvaghela** [comment]
> @sw-tt-kirtanvaghela @sw-tt-rumitkkhakhi Where is the IoTConnect 1.0 code?

IoTConnect named folder/workspace is IoTConnect 1.0 code

**Comment by sw-tt-kirtanvaghela** [comment]
> I see There are multiple folder and many files renamed please review this pull request careful and update things

Solved it

**Comment by sw-tt-kirtanvaghela** [comment]
> > @sw-tt-kirtanvaghela @sw-tt-rumitkkhakhi Where is the IoTConnect 1.0 code?
> 
> IoTConnect named folder/workspace is IoTConnect 1.0 code

**Comment by sw-tt-kirtanvaghela** [comment]
> I see There are multiple folder and many files renamed please review this pull request careful and update things

solved it

**Pull request merged**